source/slang/slang-lower-to-ir.cpp

```diff
@@ -8782,6 +8785,13 @@ struct DeclLoweringVisitor : DeclVisitor<DeclLoweringVisitor, LoweredValInfo>
             auto requirementKey = getInterfaceRequirementKey(requirementDecl);
             if (!requirementKey)
             {
+                if (auto genericDecl = as<GenericDecl>(requirementDecl))
```

Comment:
should this logic be folded into `getInterfaceRequirementKey` instead?

Reply:
This might be a bit tricky since we'll need to fold the `PropertyDecl`/`SubscriptDecl` cases below it into `getInterfaceRequirementKey` as well, and by that point, each `requirementDecl` may provide more than one key (getter + setter)